### PR TITLE
Use fog from Post-Processing Stack

### PR DESCRIPTION
**Pull request by TheLacus**
Fog component from Post Processing Stack is used, as a replacement of fog effect from old Standard Assets.
Other post processing effects are still disabled but can be enabled by mods, as it was before.

### CODE REVIEW

File: Assets/Scenes/DaggerfallUnityGame.unity [hunk 1]
```diff
@@ -753,9 +753,27 @@ Prefab:
       propertyPath: m_Enabled
       value: 1
       objectReference: {fileID: 0}
-    m_RemovedComponents: []
+    m_RemovedComponents:
+    - {fileID: 114583161180529142, guid: 53a45b46ca97a974f8e677f68c2a85e6, type: 2}
```
Comment (TheLacus) [inline Assets/Scenes/DaggerfallUnityGame.unity]: I don't know what the purpose of this is, i can't find any information.

File: Assets/Scripts/Game/UnderwaterFog.cs [hunk 2]
```diff
@@ -18,8 +14,6 @@ public class UnderwaterFog
         public UnderwaterFog()
         {
             mainCamera = GameManager.Instance.MainCamera;
-            globalFog = mainCamera.GetComponent<GlobalFog>();
-            globalFog.enabled = true;
```
Comment (TheLacus) [inline Assets/Scripts/Game/UnderwaterFog.cs]: I don't think fog is ever disabled. In that case i believe `RenderSettings.fog = true` should be the way to do it.

File: Assets/Scenes/DaggerfallUnityGame.unity (hunk 1 above)
Comment (Interkarma) [reply Assets/Scenes/DaggerfallUnityGame.unity]: I'm not sure about this one, but it's probably OK.

File: Assets/Scripts/Game/UnderwaterFog.cs (hunk 2 above)
Comment (Interkarma) [reply Assets/Scripts/Game/UnderwaterFog.cs]: That's fine. We'll need to evaluate any fog changes a bit differently now anyway.

**Pull request merged**